endpoints/cookie_sync.go

```diff
@@ -235,19 +235,22 @@ func extractPrivacyPolicies(request cookieSyncRequest, usersyncDefaultGDPRValue
 	}, gdprSignal, nil
 }
 
-func extractGDPRSignal(requestGDPR *int, gppSID []int8) (gdpr.Signal, string, error) {
-	if len(gppSID) > 0 {
-		if gppPrivacy.IsSIDInList(gppSID, gppConstants.SectionTCFEU2) {
-			return gdpr.SignalYes, strconv.Itoa(int(gdpr.SignalYes)), nil
-		}
-		return gdpr.SignalNo, strconv.Itoa(int(gdpr.SignalNo)), nil
+func extractGDPRSignal(requestGDPR *int, gppSidStr string) (gdpr.Signal, string, error) {
+
```

Comment:
Nitpick: Could you remove this blank line?

Reply:
Changes in this file were discarded